railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -56,35 +56,35 @@
   <dl>
     <% if @email.respond_to?(:smtp_envelope_from) && Array(@email.from) != Array(@email.smtp_envelope_from) %>
       <dt>SMTP-From:</dt>
-      <dd><%= @email.smtp_envelope_from %></dd>
+      <dd id="smtp-from"><%= @email.smtp_envelope_from %></dd>
```

Comment:
Maybe namespacing these to `email` to ensure their uniqueness?
I'm not sure if we should use underscores or dashes.
```suggestion
      <dd id="email-smtp-from"><%= @email.smtp_envelope_from %></dd>
```

Reply:
@p8 I was thinking about this approach but eventually decided not to do it because of existing DOM elements on this page which don't have that `email-` prefix in id attributes [here][1], [here][2] and [here][3].

Changing it everywhere and adding `email-` prefix to each element doesn't make this PR fully backwards-compatible. Let me know what do you think

[1]: https://github.com/rails/rails/blob/main/railties/lib/rails/templates/rails/mailers/email.html.erb#L102
[2]: https://github.com/rails/rails/blob/main/railties/lib/rails/templates/rails/mailers/email.html.erb#L108
[3]: https://github.com/rails/rails/blob/main/railties/lib/rails/templates/rails/mailers/email.html.erb#L116

Reply:
@p8 Should I change it to underscore? That would probably make it more consistent with existing id attributes naming e.g. https://github.com/rails/rails/blob/main/railties/lib/rails/templates/rails/mailers/email.html.erb#L108

Reply:
@p8 PR has been updated to follow existing naming convention for id attributes(underscore). Thanks for noticing the inconsistency